usrsctplib/netinet/sctp_var.h

```diff
@@ -239,9 +239,9 @@ extern struct pr_usrreqs sctp_usrreqs;
 #define sctp_free_remote_addr(__net) { \
 	if ((__net)) { \
 		if (SCTP_DECREMENT_AND_CHECK_REFCOUNT(&(__net)->ref_count)) { \
-			(void)SCTP_OS_TIMER_STOP(&(__net)->rxt_timer.timer); \
-			(void)SCTP_OS_TIMER_STOP(&(__net)->pmtu_timer.timer); \
-			(void)SCTP_OS_TIMER_STOP(&(__net)->hb_timer.timer); \
+			(void)SCTP_OS_TIMER_STOP_DRAIN(&(__net)->rxt_timer.timer); \
+			(void)SCTP_OS_TIMER_STOP_DRAIN(&(__net)->pmtu_timer.timer); \
+			(void)SCTP_OS_TIMER_STOP_DRAIN(&(__net)->hb_timer.timer); \
```

Comment:
Note for reviewers: 
I believe it is correct to use `SCTP_OS_TIMER_STOP_DRAIN`, because memory related to timers is freed just few lines below:
https://github.com/mstyura/usrsctp/blob/0e811d3eaa146a5fb3f58535638daaec41af2da9/usrsctplib/netinet/sctp_var.h#L255
where `SCTP_ZONE_FREE`:
https://github.com/mstyura/usrsctp/blob/0e811d3eaa146a5fb3f58535638daaec41af2da9/usrsctplib/netinet/sctp_os_userspace.h#L722-L725